### PR TITLE
Standalone scheme launch

### PR DESCRIPTION
**Pull request by danigm**
This PR makes possible to launch `ChannelApplication` using the scheme. The first commit make it possible to run using the default `kolibri` scheme, `gio open kolibri:3e464ee12f6a50a781cddf59147b48b1`, but there's a problem just with that change, because if the application is not running, the startup is called and the default window will be shown.

To solve that, the second commit add a new launcher application, that register the `kolibri-channel` scheme and just run the `kolibri-gnome` with the `channel-id` parameter on `open`. So it's possible to launch `ChannelApplication` without the need of a Kolibri window with `gio open kolibri-channel:3e464ee12f6a50a781cddf59147b48b1`.

In sync with this change, I've updated the desktop-xdg-plugin to use this new scheme:
https://github.com/endlessm/kolibri-app-desktop-xdg-plugin/pull/8

### CODE REVIEW

File: src/kolibri_gnome/desktop_launcher/application.py [hunk 1]
```diff
@@ -561,17 +565,35 @@ def open_url_in_window(self, url):
...
+        channel_id = url_tuple.path
+        subprocess.Popen(["kolibri-gnome", "--channel-id", channel_id])
+
+    def is_standalone(self, url):
+        url_tuple = urlsplit(url)
+        if url_tuple.query == 'standalone':
+            return True
+
+        return False
+
```
Comment (dylanmccall) [inline src/kolibri_gnome/desktop_launcher/application.py:589]: Is the set of changes from this earlier commit still relevant here? The only thing passing the `?standalone` query parameter is the desktop launchers, and if we switch those to the kolibri-channel scheme then this should all be dead code.

**Review by dylanmccall — CHANGES_REQUESTED** [review]
I like it! Good work tackling this :) One other small nitpick here is just around naming of things. I appreciate keeping the changeset simpler by not moving a dozen files around, but before merging I wonder if we should add another commit to rename desktop_launcher to just "desktop" or something, and then move `desktop_launcher/channel.py` to `launcher/application.py`?

File: src/kolibri_gnome/desktop_launcher/application.py (hunk 1 above)
Comment (danigm) [reply src/kolibri_gnome/desktop_launcher/application.py:589]: Yes, I think so, we can remove the first commit and just keep the second one with the launcher. I'll do that and I'll also add the renaming commit commented above.

**Review by dylanmccall — APPROVED** [review]
Thanks! I'm happy with this now, so I'll merge it into the add-standalone-windows branch.

**Pull request merged**